### PR TITLE
Document Apps #359: Ability to force refresh bindings for app after call

### PR DESCRIPTION
**Pull request by neflyte**
## Summary

This PR documents [Apps #359](https://github.com/mattermost/mattermost-plugin-apps/pull/359): ability to force bindings refresh after a call. It replaces #1179.

The Calls and Bindings pages were updated with new content.

### CODE REVIEW

**Comment by levb** [comment]
@neflyte sorry if I created coonfusion, but I am assuming this supersedes https://github.com/mattermost/mattermost-developer-documentation/pull/1179? If so, please close the other one. cc @asatkinson we need to streamline the process :)

**Review by asatkinson — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]

File: site/content/integrate/apps/structure/call/_index.md [hunk 1]
```diff
@@ -146,14 +146,15 @@ An example context field looks like this:
...
+| `navigate_to_url`      | string                                                                                                        | Used by the `navigate` response type to redirect the user to a specified URL.                                                  |
+| `use_external_browser` | bool                                                                                                          | Used by the `navigate` response type to indicate the system web browser should be used when redirecting the user to an URL.    |
+| `form`                 | {{<newtabref title="Form" href="https://pkg.go.dev/github.com/mattermost/mattermost-plugin-apps/apps#Form">}} | Used by the `form` response type to specify a form to display.                                                                 |
+| `refresh_bindings`     | bool                                                                                                          | If `true`, forces App bindings to be refreshed immediately. Does not apply to error responses or responses for bindings calls. |
```
[the remark this answers is not in the document]
Comment (neflyte) [reply site/content/integrate/apps/structure/call/_index.md:157]: Please elaborate on what the side-effects are of using this too often. I'd like to fully understand the impact to the customer before I adjust the wording.

[the remark this answers is not in the document]
Comment (hanzei) [reply site/content/integrate/apps/structure/call/_index.md:157]: A bindings request is a costly operation: both for the app and the apps plugin/MM server. For performance reasons, the flag should be used sparsly.

**Review by hanzei — APPROVED** [review]
LGTM :+1:

**Review by azigler — APPROVED** [review]
I agree with the feedback from @levb and @hanzei, so I added a comment to inform the reader. Looks good @neflyte !

**Comment by cwarnermm** [comment]
@asatkinson - This PR has a "do not merge/awaiting release" label. What release is this PR waiting on? Is it possible to merge this PR in now?

**Comment by cwarnermm** [comment]
Confirmed with @azigler that this PR is waiting for apps framework 1.2 to be merged and deployed into the Marketplace repo https://github.com/mattermost/mattermost-marketplace/pull/331

**Pull request merged**